src/DependencyInjection/Configuration.php

```diff
@@ -84,13 +84,23 @@ public function getConfigTreeBuilder(): TreeBuilder
                                         ->end()
                                     ->end()
                                     ->arrayNode('buckets')
-                                        ->floatPrototype()
+                                        ->floatPrototype()->end()
                                     ->end()
                                 ->end()
                             ->end()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('kv')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('rpc_dsn')->defaultValue('tcp://127.0.0.1:6001')->end()
```

Comment:
As RPC could be used for other RoadRunner plugins, I think it's worth it to move the config to the root of the bundle config.

It would require to extract the `RPC::create()` call and maybe make a re-usable RPC service.

What do you think?

Reply:
Good suggestion, upon looking in the code I noticed the bundle already has a `RPCFactory` service, I removed the config and used that one instead.